src/main/java/com/bookbla/americano/domain/school/controller/dto/response/SchoolReadResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.bookbla.americano.domain.school.controller.dto.response;
+
+import com.bookbla.americano.domain.school.repository.entity.School;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@AllArgsConstructor
```

Comment:
```suggestion
@RequiredArgsConstructor
```

모든 필드를 final로 사용하면 RequiredArgsConstructor를 사용할 수 있을 것 같아용

https://velog.io/@kitaee/AllArgsConstructor-NoArgsConstructor-RequiredArgsConstructor

Reply:
오 감사합니다! 다른 PR에서 코드 수정할게용! 😄